compiler/ast/ast.go

```diff
@@ -292,12 +292,12 @@ type (
 		Args []Assignment `json:"args"`
 	}
 
-	// An OpExprs proc is a list of assignments whose parent proc
-	// is unknown: It could be a Filter, Summarize, or Put proc. This
-	// will be determined in the semantic phase.
-	OpExprs struct {
-		Kind  string `json:"kind" unpack:""`
-		Exprs []Expr `json:"exprs"`
+	// An OpAssignments proc is a list of assignments whose parent proc
+	// is unknown: It could be a Summarize or Put proc. This will be
+	// determined in the semantic phase.
+	OpAssignments struct {
```

Comment:
I think we should call this `OpAssignment` so it's singular like all the other AST nodes.